lms/static/sass/course/modules/_student-notes.scss

```diff
@@ -68,8 +101,190 @@
   border-color: $error-color;
 }
 
-// +creating/editing notes
+// +individual note (in context)
 // --------------------
+.annotator-outer.annotator-outer {
+  @extend %ui-depth4;
+  @extend %notes-reset-font;
+}
 
-// +listing notes
+// bubble
+.annotator-widget.annotator-widget {
+  @extend %notes-bubble;
+}
+
+.annotator-item {
+  padding: ($baseline/2) !important;
+}
+
+// +creating/editing notes (overrides for vendor styling)
 // --------------------
+// adding
+.annotator-adder {
+  @extend %notes-reset-background;
+
+  button {
+    @extend %notes-bubble;
+    position: relative;
+    display: block;
+
+    &:after {
+      @extend %notes-reset-icon;
+      @extend %shame-link-base;
+      @include font-size(30);
+      position: absolute;
+      top: 35%;
+      @include left(15%);
+      content: "\f14b";
```

Comment:
NOTE: Injecting Font Awesome glyphs in pseudo elements to make controls look consistent with LMS iconography and UI styling.